healthcareai/tests/test_deploy_supervised_model_class.py

```diff
@@ -120,3 +121,22 @@ def runTest(self):
 
     def tearDown(self):
         del self.o
+
+
+class TestValidateDestinationTableConnection(unittest.TestCase):
+    def test_raises_error_on_table_not_existing(self):
+        try:
+            result = validate_destination_table_connection('localhost', 'foo', 'bar', 'baz')
+            # Fail the test if the above call doesn't throw an error
+            self.fail()
+        except HealthcareAIError as e:
+            expected_message = """Failed to insert values into foo. Check that the table exists with right column structure.
+        Your Grain ID column might not match that in your input table."""
+            self.assertEqual(e.message, expected_message)
+
+    def test_should_succeed(self):
+        is_table_connection_valid = validate_destination_table_connection('localhost',
```

Comment:
In situations like these, when indentation is already pushed pretty far to the right, I prefer this style:
```
        is_table_connection_valid = validate_destination_table_connection(
            'localhost',
            '[SAM].[dbo].[HCPyDeployRegressionBASE]',
            'PatientEncounterID',
            '[PredictedValueNBR]'
        )
```
I believe this alternative form is still pep8 compliant, but I may be wrong.